tests/api2/test_200_ftp.py

```diff
@@ -1396,6 +1423,76 @@ def setup(self, request):
             assert login_error is None
 
 
+def test_085_ftp_service_starts_after_reboot():
+    '''
+    NAS-123024
+    There is a bug in the Debian Bookwork proftpd install package
+    that enables proftpd.socket which blocks proftpd.service from starting.
+
+    We fixed this by disabling proftpd.socket. There is a different fix
+    in a Bookworm update that involves refactoring the systemd unit files.
+    '''
```

Comment:
I like this test in principle, but we probably need it disabled for now in our jenkins pipeline until we can get shifted to better infrastructure (it will be disruptive and slow things down).